template_config.yml

```diff
@@ -58,8 +58,8 @@ pulp_settings:
   allowed_import_paths:
   - /tmp
   orphan_protection_time: 0
-pulpcore_branch: 3.19
-pulpcore_pip_version_specifier: '>=3.16,<3.20'
+pulpcore_branch: 3.16
+pulpcore_pip_version_specifier: '>=3.16,<3.21'
```

Comment:
why 3.21 and not 3.25?

Reply:
I figured we can update it as we need them and don't need to do it too far in advance.

Reply:
I could bump it up another one or two releases  though, it wouldn't be an issue